publish-docs-staging.yml

```
@@ -0,0 +1,19 @@
+name: Publish docs staging
```

```suggestion
name: Publish docs
```

This flow can be used to publish docs on multiple environments (by changing `on.workflow_run.workflows`).